linera-execution/src/execution.rs

```diff
@@ -45,6 +45,14 @@ pub struct ExecutionStateView<C> {
     pub users: HashedReentrantCollectionView<C, UserApplicationId, KeyValueStoreView<C>>,
 }
 
+/// How to interact with a long-lived service runtime.
+pub struct ServiceRuntimeEndpoint {
```

Comment:
Should it be `ApplicationServiceEndpoint` ?

Reply:
I think we could do that :+1:

Reply:
Ok so there are more than names to change so this will have to wait another PR